services/ui-src/src/components/reports/DrawerReportPage.tsx

```diff
@@ -109,12 +118,13 @@ export const DrawerReportPage = ({ route, validateOnRender }: Props) => {
   };
 
   const entityRows = (entities: EntityShape[]) => {
-    return entities.map((entity) => {
+    const disabled = reportingOnIlos && !ilos?.length;
```

Comment:
Since `ilos?.length` is used a few times, could assign it to a variable `hasIlos`.